### ISSUE TITLE
allowOnlyKnownUsersToLogin and OAuth plugin

### ISSUE THREAD

**Issue by pejvan** [gocd-contrib/gocd-oauth-login#17]
Sorry mate, another question -- we should probably start an FAQ section soon :-)

Can I disable the option "Allow users that exist in LDAP or in the password file to log into Go, even if they haven't been explicitly added to Go." in the Server configuration and use the Google OAuth plug-in to get the users to log-in? Or will it prevent users from logging in if I disable this option (which seems to be the case)?  

Basically, I would like to keep this on `<security allowOnlyKnownUsersToLogin="true">` but it seems like it's not working when set to `true`.

many thanks,
Pejvan

**Comment by srinivasupadhya** [comment]
if `allowOnlyKnownUsersToLogin=true` then you will need to add every user to Go before they login
if `allowOnlyKnownUsersToLogin=false` then anyone will be able to login
see if this helps - https://github.com/srinivasupadhya/gocd-oauth-login/issues/5

**Comment by pejvan** [comment]
Many thanks for the clarification Srinivas
